lib/components/viewers/pattern-row.tsx

```diff
@@ -65,18 +68,19 @@ const PatternRow = ({
                 'gtfs'
               )}
               leg={generateFakeLegForRouteRenderer(route, true)}
-              style={{ fontSize: routeNameFontSize(routeName) }}
+              style={{ fontSize: routeNameFontSize(routeName), maxHeight: 90 }}
```

Comment:
This will cause a lot of problems. What is this for?

Reply:
<img width="578" alt="image" src="https://github.com/opentripplanner/otp-react-redux/assets/92878779/37d7aec9-59c6-49b4-872b-2c67512942fd">

Reply:
looks like it wasn't doing anything? I removed it